java/src/jmri/jmrit/display/CoordinateEdit.java

```diff
@@ -332,7 +332,7 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
     public void initSetLevel() {
         oldX = pl.getDisplayLevel();
         textX = new javax.swing.JLabel();
-        textX.setText("level= " + pl.getDisplayLevel());
+        textX.setText(Bundle.getMessage("Level") + " = " + pl.getDisplayLevel());
```

Comment:
From the experience of having localized an application before, it is much easier to handle multiple similar labels in the .properties files than it is to have partly localized Strings in code.

The entire string can be (and should be) in the .properties files, including the number formatting instructions. See the [MessageFormat JavaDocs](http://docs.oracle.com/javase/8/docs/api/java/text/MessageFormat.html) for the details of how that works.